packages/web-components/src/global/_variables.scss

```diff
@@ -0,0 +1,85 @@
+
+// Do not edit directly
+// Generated on Thu, 17 Nov 2022 19:14:09 GMT from css-library
```

Comment:
I'm not sure I understand how this file was generated. Can you document how that happened and how it made it here in this global folder?

Also how do we distinguish between `variables.css` vs `_variables.scss` in the same folder?

Reply:
### How to obtain the new tokens from `css-library`

Inside the `css-library` folder, run command `yarn build:tokens`

### Output
css
✔︎ src/tokens/_variables.css

scss
✔︎ src/tokens/_variables.scss

Reply:
> I'm not sure I understand how this file was generated. Can you document how that happened and how it made it here in this global folder?

After running the command above you will get the `_variables.css` and you can move the content to the new file in `web-components`.


> Also how do we distinguish between `variables.css` vs `_variables.scss` in the same folder?

Based on [stencil](https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/stencil.config.ts#L9-L17)

```
  globalStyle: 'src/global/main.css',
  plugins: [
    sass({ 
      includePaths: [
        '../../node_modules/@uswds/uswds/packages/',
        'src/global/'
      ],
    }),
    postcss({
      plugins: [url({ url: 'inline' })],
    }),
  ],
```
- `variables.css` gets compiled into `src/global/main.css`
- `_variables.scss` is being included only by the path. The file will need to be imported to be able to use the variables

Reply:
Thanks for outlining those steps. About `variables.css` vs `_variables.scss`,  I guess I was just wondering how a future developer would know what the difference is between them especially if that output doesn't directly map to anything (it's just manually moved at this point). 

Could you update this PR's title and/or description to specifically mention that this is a "USWDS v3" integration? While we have both v1/Formation and v3 stuff together, it can start to get difficult to distinguish things especially when they have similar names.